Tests/Integration/AbstractIntegrationTest.php

```diff
@@ -171,7 +178,7 @@ protected function handleAuthorizationRequest(ServerRequestInterface $serverRequ
             $response = $e->generateHttpResponse($response);
         }
 
-        return json_decode($response->getBody(), true);
+        return json_decode($response->getBody()->__toString(), true);
```

Comment:
@X-Coder264 Why not cast?

Reply:
@HypeMC I used it because  the `__toString` method is actually present on the `StreamInterface` PSR-7 interface.